spec/acceptance/server/grant_spec.rb

```diff
@@ -263,7 +277,7 @@ class { 'postgresql::server': }
           EOS
 
           if Gem::Version.new(postgresql_version) >= Gem::Version.new('9.0')
-            idempotent_apply(pp)
+            apply_manifest(pp, expect_changes: true)
```

Comment:
Could you please add the idempotent_apply function back since we need to test the idempotency also which applies the manifest twice.

Reply:
I have update the tests to use the idempotent_apply() function.

idempotent_apply() is done in 2 stages, because the 1st one sets up the table permissions, and the 2nd one (which uses postgresql::server::grant) 'fixes' the permissions.

When this was a single idempotent_apply() the testbench complained because postgresql::server::grant was making changes every time (as it was supposed to).

Let me know if you want further changes made to the tests.